### PR TITLE
Removed unused variables found by pylint

### PR DESCRIPTION
**Pull request by asmorkalov**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by asmorkalov** [comment]
Yes. The issues were found by modern version of pylint. Let me check the sample in details and i'll change the PR status.

**Comment by alalek** [comment]
> force-pushed the asmorkalov:as/pylint_warnings branch from a1a93e4 to 6fe90b8 yesterday
> changed the base branch from master to 3.4 yesterday

This is wrong order of steps. Branch must be changed first to trigger proper builds at Buildbot (against the right target branch).

See [here](http://pullrequest.opencv.org/buildbot/builders/precommit_linux64/builds/30763)

File: samples/dnn/virtual_try_on.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def _resample_vertical(self, out, img, ksize, bounds, kk):
             out[yy] = np.round(np.sum(img[ymin : ymin + ymax, 0:out.shape[1]] * k[:, np.newaxis], axis=0))
 
     def imaging_resample(self, img, xsize, ysize):
-        height, width, *args = img.shape
+        height, width = img.shape
```
Comment (alalek) [inline samples/dnn/virtual_try_on.py:115]: `img.shape[0:2]`?

Comment (asmorkalov) [reply samples/dnn/virtual_try_on.py:115]: Fixed.

**Pull request merged**